README.md

```diff
@@ -19,58 +19,45 @@ After cloning the smartwallet-app repository to a directory on your computer, en
 yarn install
 ```
 
-### Browser
+### Device Build
 
-#### With hot module reloading support
+#### Android
 
-```bash
-yarn start
-```
-This will make webpack-dev-server serve the application on https://localhost:8080 and will reload only the modules you've modified instead of rebuilding the entire bundle on every change. The changes will be applied live to your browser.
+Please set up an Android development environment. 
 
-#### Without hot module reloading
+Then:
 
 ```bash
-yarn build:dev
+react-native run-android
+yarn android
 ```
-Rebuilding will be much slower this way, because the entire bundle will be rebuilt on each change.
+The first command will run and install a debug build on an attached Android device. A terminal window may pop up which starts a Metro Bundler, please stop this process.
+The second command will start the Haul bundler and serve the build on your device if both are running on the same network. If remote debugging is enabled, the debugger-ui will also be available here.
 
-### Notes
+#### iOS
 
-If you are getting the error `Module parse failed: main.jsx Line 1: Unexpected token`, switch to node version 4.x before running the gulp ([nvm](https://github.com/creationix/nvm) might come in handy).
+Please set up the appropriate XCode development environment.
 
-Setup for production (iOS)
---------------------
+Then:
 
-## Installation
 ```bash
-yarn install
+react-native run-ios --device
+yarn ios
 ```
 
-### Preparing a production bundle
```

Comment:
I think preparing a production bundle should still be in the README - it's an integral part of the development process. Every new feature or bugfix should also be tested in a production build as it may reveal additional issues, just like what we had with SVG files.